myosuite/envs/myo/myochallenge/__init__.py

```diff
@@ -76,9 +97,35 @@
             'hills_range': (0.03, 0.23),
             'rough_range': (0.05, 0.1),
             'relief_range': (0.1, 0.3),
+            'repeller_opponent': False,
+            'chase_vel_range': (1.0, 1.0), # randomly drawn
+            'random_vel_range': (-2, 2), # clipped
+            'opponent_probabilities': (0.1, 0.45, 0.45),
```

Comment:
What is this and why is this of length 4 in `myoChallengeChaseTagP2eval-v0`

Reply:
this specifies the probabilities for the different opponent policies.
0.1 for stationary fixed position
0.45 for stationary random position
0.45 for random moving


There are 4 values for `myoChallengeChaseTagP2eval-v0` because there is also a repeller policy in this case.

I can add an assertion to ensure the right length with `repeller_opponent=True`

Reply:
added assertion